### DISCUSSION TITLE
Error powershell

### DISCUSSION THREAD

**Discussion by zic04** [microsoft/finops-toolkit#1091] [Q&A] [answered]
Hello,

I want to use New-FinopsCostExport and I get the following error The client does not have authorization to perform action.  (RBACAccessDenied)

    i'm owner and blob contributor
    
    Can you tell me what's missing.
    
```

New-FinopsCostExport -Name 'Daily--MTD' `
    -Scope "/subscriptions/$subclient/resourceGroups/$rgclient" `
    -StorageAccountId "/subscriptions/$storagesubid/resourceGroups/SharedStorage/providers/Microsoft.Storage/storageAccounts/$storagename" `
    -DataSet ActualCost `
    -StorageContainer $storageblob  `
    -Backfill 2 `
    -Execute
    
```

**Comment by helderpinto** [comment]
@zic04, do the Storage Account networking settings configure a firewall? If it's the case, you need to be Owner on the Storage Account ([see documentation](https://learn.microsoft.com/azure/cost-management-billing/costs/tutorial-export-acm-data?tabs=azure-portal#prerequisites)), but the Owner role must not have any conditions. If you see anything other than "None" (see example below), then this might be the reason for the error.

![image](https://github.com/user-attachments/assets/e59df9f9-2e0d-4256-a92a-9b02ae586bf2)

**Comment by zic04** [comment]
![image](https://github.com/user-attachments/assets/aab83fd9-72c7-4d78-ac96-796dc50767b9)
Rbac storage

**Comment by zic04** [comment]
```
New-FinopsCostExport -Name 'Daily--MTD' `
    -Scope "/subscriptions/$subclient" `
    -StorageAccountId "/subscriptions/$storagesubid/resourceGroups/$storageresourcegroup/providers/Microsoft.Storage/storageAccounts/$storagename" `
    -DataSet ActualCost `
    -StorageContainer $storageblob  `
    -Backfill 2 `
    -Execute
```

I have firewall and storage and I also tested without firewall and the same problem.

with  New-AzCostManagementExport i I also have the problem I'll look for

**Reply by zic04** [reply, marked as answer]
I have a problem on the network side.

Thank you.
